packages/types-algosdk/types/index.d.ts

```diff
@@ -217,10 +219,11 @@ declare module 'algosdk' {
   }
 
   export class LogicSigBase {
+    tag?: Buffer | Uint8Array;
```

Comment:
![sd](https://user-images.githubusercontent.com/26249903/109685441-44dcad80-7ba7-11eb-9fbe-72c98a186f9a.png)

I think this is not optional.

Reply:
Yes. Updating `runtime/src/logicsig.ts` as well to add `tag`.